README.md

```diff
@@ -7,15 +7,17 @@
   - [Introduction](#introduction)
     - [Before Beginning](#before-beginning)
       - [Connecting to the Correct Tenant & Setting the Correct Subscription Context](#connecting-to-the-correct-tenant--setting-the-correct-subscription-context)
-        - [PowerShell Az Modules](#powershell-az-modules)
-        - [Azure ClI](#azure-cli)
+        - [Azure ClI Login](#azure-cli-login)
   - [Setup Infrastructure](#setup-infrastructure)
   - [Infrastructure Setup During Each Script](#infrastructure-setup-during-each-script)
     - [2-CreateHub.sh](#2-createhubsh)
     - [3-AttachSpokeAndClusterToHub.sh](#3-attachspokeandclustertohubsh)
   - [Deploying NGSA Applications](#deploying-ngsa-applications)
     - [🛑 Prerequisite - Setup Cosmos DB in secure baseline](#-prerequisite---setup-cosmos-db-in-secure-baseline)
     - [Create managed identity for NGSA app](#create-managed-identity-for-ngsa-app)
+    - [Create managed identity for ngsa-app](#create-managed-identity-for-ngsa-app-1)
```

Comment:
This is a dup of the line above. Was it supposed to link somewhere else in the documentation?